src/dotnet/install.sh

```diff
@@ -194,79 +197,83 @@ apt_cache_package_and_version_soft_match() {
 }
 
 # Install .NET CLI using apt-get package installer
-install_using_apt_from_microsoft_repo() {
+install_using_apt() {
     local sdk_or_runtime="$1"
-    local dotnet_major_minor_version
-    export DOTNET_PACKAGE="dotnet-${sdk_or_runtime}"
-
-    # Install dependencies
-    check_packages apt-transport-https curl ca-certificates gnupg2 dirmngr
-
-    # Import key safely and import Microsoft apt repo
-    get_common_setting MICROSOFT_GPG_KEYS_URI
```

Comment:
Should we remove the `get_common_setting()` function as it's no longer needed?

Reply:
Fixed!